ui/src/preview/preview-table-item.tsx

```diff
@@ -218,7 +228,26 @@ export const PreviewTableItem: React.FC<PreviewTableItemProps> = ({
                             )}
                         </span>
                     </td>
-                )}
+                );
+
+            default:
+                return exhaustiveMatchingGuard(validationState);
+        }
+    })();
+
+    return (
+        <React.Fragment>
+            <tr className={'preview-table-item'}>
+                <td>{tableEntry.uiName}</td>
+                <td>{tableEntry.trackNumber ? tableEntry.trackNumber : ''}</td>
+                <td>
+                    {tableEntry.operation
+                        ? t(`enum.publish-operation.${tableEntry.operation}`)
+                        : ''}
+                </td>
+                <td>{formatDateFull(tableEntry.changeTime)}</td>
+                <td>{tableEntry.userName}</td>
+                {validationStateCell}
```

Comment:
Aika njärsyttävä. Ainoastaan yhdessä tilassa asetettiin aiemmin tuo tietty className tälle solulle (dynaamisesti). Sen voisi kyllä varmaan erottaa melko simppelisti, jolloin täälläkin olisi puhtaampi html-rakenne, as in `<td>validationStateCellValue</td>` tms

Reply:
Tuosta `validationStateCell`:istä voisi myös tehdä kokonaan oman React-komponenttinsa jolloin tässä voisi vaan kirjoittaa `<ValidationStateCell propsut={propsut} />`

Reply:
👍 Toteutettu (nappasin myös tuon riviversion tästä myös omaan komponenttiin): https://github.com/finnishtransportagency/geoviite/pull/1417